### PR TITLE
Create event by clicking on calendar

### PR DESCRIPTION
**Pull request by lkudyba**
@quirk0o @mhib @jtkpiotr @adamzima @s1mplex

### CODE REVIEW

File: client/app/components/calendar/CalendarContainer.jsx [hunk 1]
```diff
@@ -88,6 +88,13 @@ export default class CalendarContainer extends React.Component {
     this._addNotification(notification);
   }
 
+  handleAddClick(startTime, endTime) {
```
Comment (quirk0o) [inline client/app/components/calendar/CalendarContainer.jsx]: Why not handleClick or handleCellClick?

**Review by mhib — APPROVED** [review]
2/10

File: client/app/components/calendar/CalendarContainer.jsx [hunk 2]
```diff
@@ -88,6 +88,13 @@ export default class CalendarContainer extends React.Component {
     this._addNotification(notification);
   }
 
+  handleCellClick(startTime, endTime) {
+    this.openModal();
+    if (this.modal) {
```
Comment (madejejej) [inline client/app/components/calendar/CalendarContainer.jsx]: Why do you have to check for `modal` presence after opening it?

File: client/app/components/calendar/event/EventWrapper.jsx [hunk 3]
```diff
@@ -20,6 +25,7 @@ class EventWrapper extends React.Component {
 
     this.handleContainerMounted = this.handleContainerMounted.bind(this);
     this.handleResize = this.handleResize.bind(this);
+    this._handleCellClick = this._handleCellClick.bind(this);
```
Comment (madejejej) [inline client/app/components/calendar/event/EventWrapper.jsx]: How about refactoring these calls using `bindAll`?

**Comment by adamzima** [comment]
No specs :(?

**Review by madejejej — CHANGES_REQUESTED** [review]

**Review by madejejej — APPROVED** [review]

**Pull request merged**